cmd/agent/subcommands/jmx/command.go

```diff
@@ -95,7 +132,13 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 		Short: "List attributes and metrics data that match at least one of your instances configuration, including rates and counters.",
 		Long:  ``,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runJmxCommandConsole(globalParams, "list_with_rate_metrics")
+			cliParams.command = "list_with_rate_metrics"
+			disableCmdPort()
+			return fxutil.OneShot(runJmxCommandConsole,
+				fx.Supply(cliParams),
+				fx.Supply(makeCoreBundleParams(globalParams, cliParams)),
+				core.Bundle,
+			)
```

Comment:
These lines of code are the same except for `cliParams.command`. What do you think about creating a function to factorize this implementation? For example `func CreateJmxCommand(command string) func(cmd *cobra.Command, args []string) error`.